src/bitops.c

```diff
@@ -710,59 +733,144 @@ void bitopCommand(client *c) {
         if (minlen >= sizeof(unsigned long)*4 && numkeys <= 16) {
             unsigned long *lp[16];
             unsigned long *lres = (unsigned long*) res;
+            unsigned long *first_key = (unsigned long*)src[0];
+            unsigned long *multi_cnt = NULL;
+
+            size_t step = sizeof(unsigned long)*4;
+            size_t processed = 0;
 
             memcpy(lp,src,sizeof(unsigned long*)*numkeys);
-            memcpy(res,src[0],minlen);
+
+            if (op != BITOP_DIFF && op != BITOP_DIFF1 && op != BITOP_ANDOR) {
+                memcpy(lres,src[0],minlen);
+            }
```

Comment:
```suggestion
            if (op != BITOP_DIFF && op != BITOP_DIFF1 && op != BITOP_ANDOR)
                memcpy(lres,src[0],minlen);
```

I'd rather no braces here. You'll see some if checks even on same like like: 
```
if (bla) return;
```

It does not matter much for me but I think most of redis developers prefer no braces in general.